### DISCUSSION TITLE
Оформление главных страниц и их интерактивность

### DISCUSSION THREAD

**Discussion by Davud77** [totumonline/totum-ru-issues-and-discussions#165] [ОШБК!] [answered]
Всем доброго времени суток.
Хочу оформить главные страницы. Сейчас все в виде плиток при клике на которой попадаешь в таблицу. Сделал красиво добавив немного css, но хочется внести больше интерактивности, сделать так чтобы отображались не все плитки, а только те что доступны роли. Сейчас же все пользователи видят все карточки и если нет к таблице доступа куда карточка ведет получает ошибку. 

 CSS подключил через <link rel="stylesheet" href="ссылка/style.css" type="text/css"/>, а вот JS через <script src="ссылка/script.js"></script> не заработал.

![image](https://user-images.githubusercontent.com/26651156/176994238-214a1aed-d567-4ab6-a70d-c15621181d37.png)

Кроме того хотелось бы создавать кнопки при клике на которую можно создать строку.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Скорее всего, таким образом как у нас сделан там HTML, он не выполняет тег `script` — напишу, когда посмотрят и обновят версию.

**<script> разрешен с версии 3.5.44**

**Reply by Davud77** [reply]
А нет ли туториалов как создать и добавить модули пользовательские, как создавать свои интерфейсы и тд.

**Reply by ttm-alexeypolunin** [reply]
Дополнительные интерфейсы: https://ru.docs.totum.online/additional-interfaces

**Reply by ttm-alexeypolunin** [reply]
Проверьте исполнение <script> в последней версии 3.5.44
